test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -50,6 +50,8 @@
     ("aten::adaptive_avg_pool3d_backward", datetime.date(9999, 1, 1)),
     ("aten::_embedding_bag_dense_backward", datetime.date(9999, 1, 1)),
     ("aten::randperm", datetime.date(9999, 1, 1)),
+    ("aten::_masked_scale", datetime.date(9999, 1, 1)),
```

Comment:
There's no such thing as private API :( it's going to break APEX for example: https://github.com/NVIDIA/apex/search?q=_fused_dropout

perhaps keep the old wrapper for BC?

Reply:
We are okay breaking APEX in that case.  That is my code, I can fix it.